test/mocha/10-api.js

```diff
@@ -425,6 +425,53 @@ describe('api', () => {
       res2.data.authenticated.should.equal(true);
       res2.status.should.equal(200);
     });
+    it('should authenticate with a password', async function() {
+      const type = 'password';
+      const accountId = accounts['alpha@example.com'].account.id;
+      const password = 'test-password';
+      const saltRounds = 10;
+      const hash = await bcrypt.hash(password, saltRounds);
+      const {agent} = brHttpsAgent;
```

Comment:
Just a note that with bedrock-https-agent@2, this can now be moved to the header along with other imports/requires.

Reply:
Updated here cc8f8e0